setup.py

```diff
@@ -80,9 +80,9 @@
 ]
 
 extra_deps['gpu'] = [
-    'flash-attn==v1.0.3.post0',
+    'flash-attn==v1.0.7.post0',
```

Comment:
```suggestion
    'flash-attn==v1.0.7',
```

I dont think `v1.0.7.post0` [exists](https://github.com/HazyResearch/flash-attention/tags)